content/docs/2.13/authentication-providers/aws-eks.md

```diff
@@ -4,6 +4,8 @@ title = "EKS Pod Identity Webhook for AWS"
 
 [**EKS Pod Identity Webhook**](https://github.com/aws/amazon-eks-pod-identity-webhook), which is described more in depth [here](https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/), allows you to provide the role name using an annotation on a service account associated with your pod.
 
+> ⚠️ **WARNING:** `aws-eks` auth has been deprecated and support for it will be removed from KEDA on v3. We strongly encourage the migration to [`aws` auth](./aws.md).
```

Comment:
There isn't any discussion because we didn't merge the feature yet, IIRC the discussion has to be created during the release process, hasn't?
I've already created the issues, I can link them (or I can create the discussion now and add them)

Reply:
```suggestion
> ⚠️ **WARNING:** [`aws-eks` auth has been deprecated](https://github.com/kedacore/keda/issues/5086) and support for it will be removed from KEDA on v3. We strongly encourage the migration to [`aws` auth](./aws.md).
```

Reply:
> There isn't any discussion because we didn't merge the feature yet, IIRC the discussion has to be created during the release process, hasn't?

Not specifically, as part of the deprecation as that is sooner

Reply:
Okey, I miss understood the process and I haven't created them yet, only the issues.
Let me create them too and update the links

Reply:
done